src/v2/openapi.yaml

```diff
@@ -0,0 +1,740 @@
+        - current_stack_license
+        - unknown_licenses
+      properties:
+        outlier_packages:
+          type: array
+          items:
+            type: object
+            properties: {}
+        conflict_packages:
+          type: array
+          items:
+            $ref: '#/components/schemas/ConflictPackages'
+        current_stack_license:
+          type: object
+          properties: {}
+        unknown_licenses:
+          $ref: '#/components/schemas/UnknownLicenses'
+      description: License analyses data
+
+    GitHubDetails:
+      title: GitHubDetails
+      type: object
+      properties:
+        watchers:
+          type: integer
+          example: 1002
+        first_release_date:
+          type: string
+        total_releases:
+          type: integer
+          example: 12
+        issues:
+          type: object
+          properties: {}
+        pull_requests:
+          type: object
+          properties: {}
+        dependent_repos:
+          type: integer
+        open_issues_count:
+          type: integer
+        latest_release_duration:
+          type: string
+        forks_count:
+          type: integer
+          example: 123
+        contributors:
+          type: integer
+          example: 12
+        size:
+          type: string
+        stargazers_count:
+          type: integer
+          example: 50000
+        used_by:
+          type: array
+          items:
+            type: string
+        dependent_projects:
+          type: integer
+          example: 1534
+      description: Github details
+
+    PackageDetails:
+      title: PackageDetails
+      allOf:
+        - $ref: '#/components/schemas/Package'
+        - type: object
+          required:
+            - latest_version
+            - ecosystem
+            - public_vulnerabilities_count
+            - private_vulnerabilities_count
+          properties:
+            latest_version:
+              type: string
+              example: 3.0.5
+            github:
+              $ref: '#/components/schemas/GitHubDetails'
+            licenses:
+              type: array
+              items:
+                type: string
+              example:
+                - Apache License, Version 2.0
+            topic_list:
+              type: array
+              items:
+                type: string
+            ecosystem:
+              $ref: '#/components/schemas/Ecosystem'
+            public_vulnerabilities_count:
+              type: integer
+              example: 21
+            private_vulnerabilities_count:
+              type: integer
+              example: 3
+            osio_user_count:
+              type: integer
+              example: 123
+            url:
+              type: string
+              example: 
+                - https://snyk.io/vuln/pip:django
+                - https://snyk.io/vuln/pip:jinja2
+          description: Details of the package
+
+    RecommendedVersionForPublicVulnerabilities:
```

Comment:
Do we need an object with one field. Can this not directly be added ?

Reply:
It clearly puts the fact in the title that there will two different recommendations. It can be a direct field inside the `PackageDetails` model, but we need to add proper description.

Reply:
This can be added as a property in the PackageDetailForRegisteredUser. That will make it clear. I don't think we need 2 properties to talk about the same data.